src/Instance.ts

```diff
@@ -198,6 +200,19 @@ export default class Instance {
     return this.connection;
   }
 
+  async getLibraryList(connection: IBMi, workspaceFolder?: vscode.WorkspaceFolder): Promise<ILELibrarySettings> {
+    const config = connection.getConfig();
+
+    const env = workspaceFolder ? (await getEnvConfig(workspaceFolder)) : {};
+
+    const librarySetup: ILELibrarySettings = {
+      currentLibrary: env[`&CURLIB`] || config.currentLibrary,
+      libraryList: env[`&LIBL`]?.split(` `) || config.libraryList,
+    };
+
```

Comment:
Should we remove duplicates like we do in `CompileTools` or leave that up to the caller?
```suggestion
librarySetup.libraryList.filter(Tools.distinct)
```

Reply:
Yup, let's leave it to the caller!